### DISCUSSION TITLE
When SELECT query, it shows data but when CREATE TABLE AS SELECT , shows empty

### DISCUSSION THREAD

**Discussion by dinggege1024** [apache/doris#29950] [RFC]
SELECT查询有数据，CREATE TABLE AS SELECT数据为空
![image](https://github.com/apache/doris/assets/109070189/edd6df92-a606-4151-a46c-11d960068535)
create table tmp_res_ma75_hijacked_role as
SELECT
    role_id,
    udid,
    before_appset,
    after_appset,
    ARRAY_EXCEPT(before_appset,ARRAY_INTERSECT(before_appset,after_appset)) as before_expect_appset,
    ARRAY_EXCEPT(after_appset,ARRAY_INTERSECT(before_appset,after_appset)) as after_expect_appset,
    ARRAY_INTERSECT(before_appset,after_appset) as common_app
FROM
    tmp_ma75_hijacked_role;

**Comment by dinggege1024** [comment]
doris 1.2.6

**Comment by LemonLiTree** [comment]
Is it okay to create the table first and then insert into it?

**Comment by dinggege1024** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    Yes, It is still unavailable.
    
        
            
        
    
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                dinggege1024
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
 
    

  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2024年01月16日 11:41
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [apache/doris] SELECT查询有数据，CREATE TABLE AS SELECT数据为空 (Discussion #29950)
        
      
    
  
  
Is it okay to create the table first and then insert into it?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
